tasks/fix-cat2.yml

```diff
@@ -711,12 +711,22 @@
       - RHEL-07-020650
 
 - name: "MEDIUM | RHEL-07-020660 | PATCH | All files and directories contained in local interactive user home directories must be owned by the owner of the home directory."
-  command: "true"
-  changed_when: no
+  block:
+    - name: "MEDIUM | RHEL-07-020660 | AUDIT | All files and directories contained in local interactive user home directories must be owned by the owner of the home directory."
+      command: find {{ item.split(':')[2] }} ! ( -user {{ item.split(':')[0] }} -o -user root )
+      changed_when: no
+      register: rhel_07_020660_audit
+      with_items: "{{ rhel7stig_passwd_homedir_audit.stdout_lines }}"
+
+    - name: "MEDIUM | RHEL-07-020660 | PATCH | All files and directories contained in local interactive user home directories must be owned by the owner of the home directory."
```

Comment:
Fine to change this to a 'find -exec' or 'find | xargs -0'.  Condition its run on:
```
when: rhel_07_020660_audit | changed
```